open-apparel-registry/open-apparel-registry
# Add django-simple-history tracking to FacilityMatch

## Overview

To support deleting facilities and fixing matching mistakes we want to track
edits and deletions of `FacilityMatch` records.

Connects #604

## Demo

<img width="1195" alt="Screen Shot 2019-06-17 at 9 57 15 AM" src="https://user-images.githubusercontent.com/17363/59622416-571e6b80-90e6-11e9-8d39-a87b0515f88f.png">

## Testing Instructions

This assumes you have fixture data loaded and processed

* Setup and run
  * `./scripts/setup`
  * `./scripts/manage populate_history --auto`
  * `./scripts/server`
* Browse http://localhost:8081/admin/api/facilitymatch/914/history/ and verify that the django-simple-history view is displayed.

## Checklist

- [x] ~CHANGELOG.md updated with summary of features or fixes, following [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) guidelines~ Skipped because this is an internal change.
